koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -58,6 +60,8 @@
 else:
     osbs_flatpak_support = True
 
+BUILD_PARAMS_SCHEMA_PATH = '/usr/share/koji-containerbuild/schemas/build_params.json'
```

Comment:
Nice! this seems to be the proper place for the file.

Should the python package also install it here? I mean, what happens if this package is installed with `pip install`? You can take a look at how we do it in atomic-reactor for reference (check the `setup.py` file).

Reply:
`/usr` part cannot be hardcoded, `pip install --user` or installation into virtualenv places `share` directory to somewhere else. This part mus be detected dynamically.